src/main/java/nl/tudelft/pixelperfect/event/EventParameter.java

```diff
@@ -0,0 +1,72 @@
+package nl.tudelft.pixelperfect.event;
+
+/**
+ * Class representing a single parameter of an Event, having either a discrete value
+ * (EventParameterValue) or a continuous one (Integer).
+ * 
+ * @author Jesse Tilro
+ *
+ */
+public class EventParameter {
+
+  private String key;
+  private EventParameterValues value;
+  private int number;
```

Comment:
Yes, but the naming is rather useful for future methods. For example, what if I am in a method and it just says 'number', I would have to backtrack (ofc, that would not be hard here).

It is just a little bit of clarification of an important variable, it is not a fatal error.